src/pkg/services/m365/api/drive_pager.go

```diff
@@ -21,7 +21,7 @@ import (
 // item pager
 // ---------------------------------------------------------------------------
 
-type driveItemPager struct {
+type DriveItemPager struct {
```

Comment:
nit: Add `var _ DriveItemEnumerator = &DriveItemPager{}` (and others) in this file too?